src/Equinor.ProCoSys.IPO.Command/InvitationCommands/CreateInvitation/CreateInvitationCommandHandler.cs

```diff
@@ -117,6 +125,10 @@ public async Task<Result<int>> Handle(CreateInvitationCommand request, Cancellat
             meetingParticipants = await AddParticipantsAsync(invitation, meetingParticipants, request.Participants.ToList());
         
             await _unitOfWork.SaveChangesAsync(cancellationToken);
+            
+            //Manual publish instead of domain event handler in order to obtain the id of the invitation
+            await PublishEventToBusAsync(invitation, cancellationToken);
```

Comment:
unless we wrap the whole handler in a transaction, we may get inconsistencies when we call this after SaveChangesAsync?

Reply:
The handler is wrapped in a transaction I think??

Reply:
it is too